<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XUnitInputDialog</class>
 <widget class="QDialog" name="XUnitInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set X axis unit ratio</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="ql_message">
     <property name="text">
      <string>&lt;PLACEHOLDER&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ql_messagePartTwo">
     <property name="text">
      <string>Please provide the conversion ratio of the X axis unit to seconds manually.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ql_second">
       <property name="text">
        <string>[second] = </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qle_multiplier"/>
     </item>
     <item>
      <widget class="QLabel" name="ql_multiply">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ql_unit">
       <property name="text">
        <string>&lt;UNIT&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
